src/python/ServerUtilities.py

```diff
@@ -951,3 +951,25 @@ def get_size(obj, seen=None):
     elif hasattr(obj, '__iter__') and not isinstance(obj, (str, bytes, bytearray)):
         size += sum([get_size(i, seen) for i in obj])
     return size
+
+
+def parseDBSInstance(dbsurl):
+    """Parse `tm_dbs_url` to get dbsInstance in `<dbsenv>/<instance>` format.
+
+    >>> url = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'
+    >>> parseDBSInstance(url)
+    'prod/global'
+    """
+    return "{}/{}".format(dbsurl.split("//")[1].split("/")[2], dbsurl.split("//")[1].split("/")[3])
```

Comment:
Sorry If i jump in. I think this requires the implicit agreement that we and users will always spell the urls correctly (crab config has the `Data.inputDBS` argument). If we throw in an extra slash, this will break. I would suggest using 

```python
from urllib.parse import urlparse
import re

path = urlparse(dbsurl).path
path = re.sub("/{2,}", "/", path)

return "{}/{}".format(path.split("/")[2], path.split("/")[3])
```

Maybe I am always a bit too defensive, but I dont really like manually parsing urls, weird things can happen, i'd rather use urlparse as much as possible.

Reply:
that URL is sanitized when entered in CRAB's DB. Here we can use it confidently.

Reply:
that said, urlparse surely is cool way to parse URL's. Personally I understand that ugly code I wrote, but I (and others) always get lost in regexps. Anyhow this is about code which predates current PR. Dario feel free to make a PR to improve, but please something which even  I can tell that it is correct, maybe it is enough to add some terse comments ?